src/core/util/prefixer.js

```diff
@@ -0,0 +1,63 @@
+angular
+  .module('material.core')
+  .config( function($provide) {
+    $provide.decorator('$mdUtil', ['$delegate', function ($delegate) {
+
+      // Inject the prefixer into our original $mdUtil service.
+      $delegate.prefixer = MdPrefixer;

```

Comment:
Hm, I don't really like it.

Then the actual `prefixer()` will be only used for `hasAttribute`. 
This will make the whole constructor function unnecessary.

I would rather change the constructor fn to something like that
```js
$mdUtil.prefixer('ng-click', true)
```